src/uct/api/v2/uct_v2.h

```diff
@@ -963,6 +976,31 @@ ucs_status_t uct_ep_connect_to_ep_v2(uct_ep_h ep,
                                      const uct_ep_addr_t *ep_addr,
                                      const uct_ep_connect_to_ep_params_t *params);
 
+
+/**
+ * @ingroup UCT_MD
+ *
+ * @brief Compare two remote keys
```

Comment:
this is not a sentence

Reply:
replaced by `This routine compares two remote keys.`